packages/fx-core/tests/component/driver/aad/create.test.ts

```diff
@@ -327,7 +327,9 @@ describe("aadAppCreate", async () => {
     expect(result.result._unsafeUnwrapErr())
       .is.instanceOf(UnhandledUserError)
       .and.has.property("message")
-      .and.contains("An unexpected error has occurred while performing the aadApp/create task");
+      .and.equals(
+        'An unexpected error has occurred while performing the aadApp/create task. {"error":{"code":"Request_BadRequest","message":"Invalid value specified for property \'displayName\' of resource \'Application\'."}}'
```

Comment:
@jidddddd can you help review the new message? The API response various in different situation, so we put the raw response here to avoid missing useful information.